IFC4x3/PropertyConstants/c/COMPOUNDCATENARYSUSPENSION_0kx1aFr$H7sQ0QknW4mZF7.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="COMPOUNDCATENARYSUSPENSION_0kx1aFrH7sQ0QknW4mZF7" Name="COMPOUNDCATENARYSUSPENSION" UniqueId="2eec190f-d7f4-47d9-a01a-bb1804c233c7" />
```

Comment:
One could be deleted.

Reply:
Done: https://github.com/bSI-InfraRoom/IFC-Specification/pull/190/commits/be731fb1169a876200741df2c81db4c91e730337